packages/front-end/components/Settings/Webhooks.tsx

```diff
@@ -150,7 +159,10 @@ const Webhooks: FC = () => {
           </tbody>
         </table>
       )}
-      {permissionsUtil.canCreateSDKWebhook() ? (
+      {permissionsUtil.canCreateSDKWebhook({
+        projects: [project],
+        environment: "",
+      }) ? (
```

Comment:
This isn't technically correct - but given this [PR](https://github.com/growthbook/growthbook/pull/2483/files#diff-9b0f37869e5ee3a773f1f808d099594cf2d7c9eea5fe7a82f4478092074d1449) will remove this option, I haven't fixed it.